README.md

```diff
@@ -30,4 +30,21 @@ npm install
 bin/rails db:create db:schema:load
 bin/rails db:seed
 bin/rails s
-```
\ No newline at end of file
+```
+
+## Overrides, decorators and more.
+
+### Add Google Tag Manager to admin backoffice
+
+Deface has a strange fail with nokogiri gem and the overrides with Deface not works with this changes.
```

Comment:
Can you write the required `nokogiri` version in order to use deface in the HEAD tag please?